paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -910,6 +910,18 @@ def valid_transitions(self) -> Iterator[state_machine.TransitionDefinition]:
                 "dest": None,  # this makes it an "internal transition", effectively a noop.
                 "trigger": "rollback_slo_failure",
             }
+            yield {
+                "source": self.rollforward_states,
+                "dest": "start_rollback",
+                "trigger": "rollback_metric_failure",
+                "before": self.log_metric_rollback,
+            }
+            # TODO COMPINFRA-1140 - Is this transition necessary? Internal transition with no before/after/prepare does...nothing?
+            yield {
```

Comment:
Might be worth leaving this out for now and only adding if we notice missing functionality. I feel like the SLO one was added incorrectly, based on my understanding of Transitions